cristin-project/src/main/java/no/unit/nva/cristin/projects/FetchCristinProjects.java

```diff
@@ -59,9 +61,9 @@ protected FetchCristinProjects(CristinApiClient cristinApiClient, Environment en
     }
 
     @Override
-    protected void validateQueryParamKeys(RequestInfo requestInfo) throws BadRequestException {
-        if (!VALID_QUERY_PARAMS.containsAll(requestInfo.getQueryParameters().keySet())) {
-            throw new BadRequestException(ERROR_MESSAGE_INVALID_QUERY_PARAMS_ON_SEARCH);
+    protected void validateQueryParameterKeys(RequestInfo requestInfo) throws BadRequestException {
+        if (!VALID_QUERY_PARAMETERS.containsAll(requestInfo.getQueryParameters().keySet())) {
+            throw new BadRequestException(ERROR_MESSAGE_INVALID_QUERY_PARAMETERS_ON_SEARCH);
```

Comment:
In a new PR, can you ensure that we replace this string with a template string, where we input the set of valid parameters — in all places that we do this?